content/pages/auth/login.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Welcome back
+description: Login to access all the features on TrollTracker
+signupPrompt: New to Trolltracker?
+signUpLink: Sign up now
+href: /signup
+password: Forgot password?
+googleIcon: /images/GoogleIcon.svg
```

Comment:
Google Icon is not visible on preview link. This could be the reason.. the google image added in `public/images` is `googleicon.svg` not `GoogleIcon.svg`